Contest_API.md

```diff
@@ -1062,15 +1062,25 @@ To add submissions one can use the `POST` method on the submissions endpoint.
 The `POST` must include a valid JSON object with the same attributes the submission
 endpoint returns with a `GET` request with the following exceptions:
 
-* `id`, `reaction` and `contest_time` should not be provided, and are ignored.
+* `reaction` and `contest_time` should not be provided, and are ignored.
 * The `time` attribute is optional. If not provided (or `null`) it will default
   to the current time as determined by the server.
 * Since `files` only supports `application/zip`, providing the `mime` field is
   optional.
-* If the CCS supports a `team` role, the `team_id` and `time`
+* If the CCS supports a `team` role, the `team_id`, `time` and `id`
   attributes will be ignored when using this role. `team_id` will then be set to
   the ID of the team associated with the request and `time` will always use
-  the current time as determined by the server.
+  the current time as determined by the server. The CCS will determine an `id`.
+* If an `id` is supplied, the client should make sure it is unique, i.e. not used
+  yet on the CCS.
```

Comment:
I'd add here something along the lines of:
The client should normally not supply `id`, but let it be determined by the server. However, for example in a setup with a central CCS with satellite sites where teams submit to a proxy CCS and that forwards to the central CCS, this might be useful to make sure that the proxy CCS can accept submissions even when the connection to the central CCS is down. Then the proxy can forward these submissions later, when the connection is restored again.